### DISCUSSION TITLE
Voice Mail FWD

### DISCUSSION THREAD

**Discussion by FabrizioMoggio** [camaraproject/CallForwardingSignal#4] [Ideas]
Potential Requirement: redirection to voice mail must considered as valid?

**Comment by bigludo7** [comment]
Humm good question; I will probably park it for later consideration as probably our main priority is unconditional forwarding to another number.

**Comment by FabrizioMoggio** [comment]
The information on the voiceMail could be useful for: https://github.com/camaraproject/CallForwardingSignal/discussions/31

Anyway this discussion will not limit the Alpha release.
